### DISCUSSION TITLE
Which graph file to use?

### DISCUSSION THREAD

**Discussion by felince** [ablab/spades#1321] [Q&A]
Hi,

I am attempting to assemble some plasmids from hybrid PacBio and DNBseq data and I by looking at the output files I obtained two .gfa files so I wonder which one represents the finished assembly?

assembly_graph_with_scaffolds.gfa or assembly_graph_after_simplification.gfa?

also, what does this "after simplification" mean? can't find it in the manual.

Thanks

**Comment by asl** [comment]
Per manual https://ablab.github.io/spades/output.html, it is `assembly_graph_with_scaffolds.gfa` that you should use

**Comment by felince** [comment]
Thank you. Any idea about the meaning of the after_simplification file?
